apps/server/src/modules/deletion/controller/dto/deletion-request.body.params.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { IsNumber, IsOptional, Min } from 'class-validator';
+import { DeletionTargetRef } from '../../uc/interface';
+
+export class DeletionRequestBodyProps {
+	@ApiProperty({
+		required: true,
+		nullable: false,
+	})
+	targetRef!: DeletionTargetRef;
+
+	@IsNumber()
+	@Min(0)
+	@IsOptional()
+	@ApiPropertyOptional({
+		required: true,
+		nullable: false,
+	})
+	deleteInMinutes?: number = 43200;
```

Comment:
Can you please add on top a const MINUTS_OF_30_DAYS = 24*60*60 and use it with number = MINUTS_OF_30_DAYS ;

Reply:
I can but I think value you wrote is bad 30 days in minutes equals 43200

Reply:
done